mkat_fpga_tests/test_cbf.py

```diff
@@ -59,18 +59,23 @@ def setUp(self):
         self.addCleanup(self.corr_fix.stop_x_data)
         self.corr_fix.start_x_data()
         self.corr_fix.issue_metadata()
+        #test_chan=1500 # possibly make this a global.
+        # Threshold: -70dB
+        self.threshold=1e-7
 
     # TODO 2015-05-27 (NM) Do test using get_vacc_offset(test_dump['xeng_raw']) to see if
     # the VACC is rotated. Run this test first so that we know immediately that other
     # tests will be b0rked.
-
     def test_channelisation(self):
         """TP.C.1.19 CBF Channelisation Wideband Coarse L-band"""
-        test_chan = 1500
-        expected_fc = self.corr_freqs.chan_freqs[test_chan]
         test_name = '{}.{}'.format(strclass(self.__class__), self._testMethodName)
         test_data_h5 = TestDataH5(test_name + '.h5')
         self.addCleanup(test_data_h5.close)
+        test_chan=1500
```

Comment:
Here and several other place, you must put spaces around and `=` used for assigment. No spaces around `=` is only for keyword arguments.